.requirements

```diff
@@ -12,3 +12,7 @@ LUA_RESTY_WEBSOCKET=60eafc3d7153bceb16e6327074e0afc3d94b1316 # 0.4.0
 ATC_ROUTER=72cc8fddeac024c54c9c1fa5a25c28a72d79080e # 1.1.0
 
 KONG_MANAGER=nightly
+NGX_WASM_MODULE_BRANCH=main
+WASMER_VERSION=3.1.1
+WASMTIME_VERSION=8.0.1
+V8_VERSION=10.5.18
```

Comment:
Also, the current version the module is tested for today is `11.4.183.23`

Reply:
This can be addressed when we will release ngx_wasm_module `prerelease-0.1.0`, which will pin the supported runtime versions.